components/brave_wallet_ui/constants/types.ts

```diff
@@ -673,7 +673,10 @@ export enum CoinTypesMap {
 
 export type BuyOption = {
   id: BraveWallet.OnRampProvider
-  label: string
+  icon: any
```

Comment:
Should this be a ImageSource or ReactNode?

Reply:
I have changed this to `string`. Image `src` are essentially strings.